Sources/Overlays/_Testing_CoreImage/ReexportTesting.swift

```diff
@@ -0,0 +1,12 @@
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2024 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+//
+
+@_exported @_spi(Experimental) @_spi(ForToolsIntegrationOnly) public import Testing
+@_exported @_spi(Experimental) @_spi(ForToolsIntegrationOnly) public import _Testing_CoreGraphics
```

Comment:
Do the `@_spi` on these re-exports actually have observable effect in client modules?